frida_tools/application.py

```diff
@@ -445,6 +451,47 @@ def work():
 
         return result[0]
 
+    def return_real_args(self, parser):
+        real_args = sys.argv[1:]
+        performPass = True
```

Comment:
Variable names should not be camelCase, but snake_case. (We're following PEP-8 with the addition of double- vs. single-quotes for strings vs. enums.)

Reply:
Done